ur_bringup/launch/ur3.launch.py

```diff
@@ -26,7 +26,6 @@ def generate_launch_description():
     declared_arguments.append(
         DeclareLaunchArgument(
             "robot_ip",
-            default_value="",
```

Comment:
Nit: this type of change would be better in a separate PR

Reply:
I would keep those here. It doesn't make much sense to remove them now. But yeah I agree, this PR become huge... It's just that many changes were needed and time is limited :)